src/components/Setting/Context/Provider.js

```diff
@@ -46,4 +46,11 @@ export const useSettingDispatchCtx = () => {
   return dispatch
 }
 
+export const useSettingContext = () => {
```

Comment:
I am not quite familiar with this context pattern,
just curious should we rename one of the `useSettingCtx`, `useSettingDispatchCtx` or `useSettingContext` to make more sense?

Reply:
I think we should rename `useSettingCtx` to `useSettingStateCtx` and `useSettingContext` to `useSettingCtx` whatever just make sure the naming rule is aligned.
https://kentcdodds.com/blog/how-to-optimize-your-context-value